docs/security/password-protection.mdx

```diff
@@ -17,15 +17,39 @@ While these rules might seem lax independently, NIST's additional leaked passwor
 
 Also, please bear in mind, that passwords are not a requirement for using Clerk. Applications can be configured to use a passwordless strategy that relies on your users being sent one-time passwords instead.
 
-## Leaked password protection
-Clerk refers to the National Institute of Standards and Technology (NIST) guidelines to determine its handling of leaked passwords:
+## Reject compromised passwords
+Clerk refers to the National Institute of Standards and Technology (NIST) guidelines to determine its handling of compromised passwords:
 
 When processing requests to establish and change memorized secrets, verifiers SHALL compare the prospective secrets against a list that contains values known to be commonly-used, expected, or compromised. For example, the list MAY include, but is not limited to: [NIST Special Publication 800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html#sec5)
 
 - Passwords obtained from previous breach corpuses.
 
 Specifically, Clerk contracts with have [i been pwned](https://haveibeenpwned.com/) to compare prospective passwords against its corpus of over 10 billion leaked credentials.
 
+Rejection of compromised passwords is enabled by default on all Clerk apps, for sign-up and password changes.
+
+Settings for this feature can be found in the Clerk Dashboard under **User & Authentication > [Email, Phone, Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)**, in the **Authentication strategies > Password** section.
+
+![Settings for compromised password handling](/public/images/authentication/reject_compromised_passwords.png)
```

Comment:
```suggestion
```

In general, we avoid screenshots unless they're required for understanding the information being presented

Reply:
Removing